### ISSUE TITLE
Support xhtml-3000.3.0.0

### ISSUE THREAD

**Issue by Bodigrim** [Happstack/happstack-server#85]
...and add a missing upper bound in the meantime, otherwise:
```haskell
$ cabal build -c 'xhtml >= 3000.3.0.0'
Build profile: -w ghc-9.6.5 -O1
In order, the following will be built (use -v for more details):
 - happstack-server-7.9.0 (lib) (first run)
Preprocessing library for happstack-server-7.9.0...
Building library for happstack-server-7.9.0...
[18 of 35] Compiling Happstack.Server.Response [Text.XHtml package changed]

src/Happstack/Server/Response.hs:143:33: error: [GHC-83865]
    • Couldn't match type ‘Data.ByteString.Builder.Internal.Builder’
                     with ‘[Char]’
      Expected: XHtml.Html -> String
        Actual: XHtml.Html -> Data.ByteString.Builder.Internal.Builder
    • In the second argument of ‘(.)’, namely ‘XHtml.renderHtml’
      In the expression: LU.fromString . XHtml.renderHtml
      In an equation for ‘toMessage’:
          toMessage = LU.fromString . XHtml.renderHtml
    |
143 |     toMessage = LU.fromString . XHtml.renderHtml
    |                                 ^^^^^^^^^^^^^^^^
Error: [Cabal-7125]
Failed to build happstack-server-7.9.0.
```

**Comment by Bodigrim** [comment]
As a Hackage trustee I revised the latest releases to exclude unbuildable configurations:
https://hackage.haskell.org/package/happstack-server-7.9.0/revisions/
https://hackage.haskell.org/package/happstack-server-7.8.0.2/revisions/
https://hackage.haskell.org/package/happstack-server-7.8.0.1/revisions/
https://hackage.haskell.org/package/happstack-server-7.8.0/revisions/
https://hackage.haskell.org/package/happstack-server-7.7.2/revisions/
https://hackage.haskell.org/package/happstack-server-7.7.1.2/revisions/
https://hackage.haskell.org/package/happstack-server-7.7.1.1/revisions/
https://hackage.haskell.org/package/happstack-server-7.7.1/revisions/

**Comment by stepcut** [comment]
Fixed in 7.9.2
